bookops_worldcat/metadata_api.py

```diff
@@ -50,6 +50,9 @@ def _split_into_legal_volume(
         for i in range(0, len(oclc_numbers), n):
             yield ",".join(oclc_numbers[i : i + n])  # noqa: E203
 
+    def URL_BASE(self) -> str:
+        return "https://metadata.api.oclc.org/worldcat"
+
     def _url_base(self) -> str:
```

Comment:
Am I correct this will be eventually eliminated and this `_url_base` in not valid for the v2.0 of the API?

Reply:
Adding `URL_BASE` while leaving `_url_base` and `_url_search_base` was my attempt to make changes in these PRs easier to read and merge. Adding and removing a lot of lines made it pretty hard to view the changes when I submitted everything in one bit PR.

I'll remove `_url_base` and `_url_search_base` and do some more cleanup in a future PR.

Reply:
Great! Thanks for the explanation.